app/views/terms/show.html.erb

```diff
@@ -22,30 +22,23 @@
 <% if @term.term_id.hasParent? %>
   <%= link_to "Return to Vocabulary", term_path(:id => @term.term_uri_vocabulary_id)  %>
 <% end %>
-<table class="table table-condensed" style="margin-top: 20px;">
-  <% @term.fields.each do |field| %>
-    <tr>
-      <th scope="row"><%= field.to_s.humanize %>:</th>
-      <td>
-        <ul style="list-style: none;">
-          <% if @term.blacklisted_language_properties.include?(field) %>
-            <% @term.get_values(field).each do |t| %>
-              <li><%= t %></li>
-            <% end %>
-          <% else %>
-            <% @term.literal_language_list_for_property(field).each do |t| %>
-              <% if t[1] == "Language Not Found" %>
-                <li><%= "#{t[0].value}" %> </li>
-              <% else %>
-                <li><%= "#{t[0].value} (#{t[1]}/#{t[0].language})" %></li>
-              <% end %>
-            <% end %>
-          <% end %>
-        </ul>
-      </td>
-    </tr>
-  <% end %>
-</table>
+<% if @term.vocabulary? %>
+  <table class="table table-condensed" style="margin-top: 20px;">
+    <%= render :partial => "terms/term_row", locals: {:field => :title} %>
```

Comment:
The issue is that it inherits the fields in order from the term fields first then appends the vocabulary fields at the end since vocabulary is a subclass of term. I could add a new method to vocabulary, something like #vocabulary_fields, which is an ordered list of all the fields in the order we want.